proposals/stage-1/TRAVEL_FUND/MEMBER_TRAVEL_FUND.md

```diff
@@ -0,0 +1,169 @@
+guaranteed. If a request is rejected, reimbursement will not be issued.The Foundation cannot pre-pay for travel expenditures.
+It is recommended to write a report after the trip.
+
+## Visa Tips
+
+For all events, 3rd party or hosted Foundation events, it is recommended to apply for a visa at least 6 months prior to an event.
+
+### For Industry Events
+
+If you need a visa for the country in which the OpenJSF event you are attending is being held,
+[see the document on preparing for a visa application](https://github.com/nodejs/admin/blob/master/travel-visas.md). //TODO update link
+
+### What The Foundation Can Do To Help:
+
+//TODO update link
+The Foundation [provides guidance on preparing your visa application](https://github.com/nodejs/admin/blob/master/travel-visas.md),
+including information on visa processing times and a checklist of items that a visa application could require.
+
+For Foundation-hosted events, OpenJS Foundation provides a visa letter by The Linux Foundation (does not guarantee visa approval).
+Click [here](https://events.linuxfoundation.org/events/node-js-interactive-2018/attend/visa-request/) to request a visa letter.
+
+### What the Foundation Cannot Do:
+
+Please note that unless the Foundation is specifically hosting the event, we cannot help with visa letters.
+The Foundation is also not able to contact consulates. When the Foundation has done this in the past, our efforts have been unsuccessful
+and/or we’ve lacked the expertise to be effective.  
+
+For additional information on preparing your visa application or requesting a visa letter, please email
+[visainfo@openjsf.org](mailto:visainfo@openjsf.org).
+
+## Requesting Travel Funds
+
+* Open a pull request that adds an entry to the appropriate year's file
+  * Include as many fields as possible:
+    * The name of the event you plan to attend.
+    * The location, dates of the event and where you will be traveling from.
+    * The presentation you intend to give, if applicable.
+    * The size of the stipend you wish to receive.
+      * In the pull request description, explain how you calculated your
+        stipend. For example, include estimates for airfare, hotel, other
+        transportation, and so on. The total amount should be your stipend
+        request.
+      * In the PR description, mention @openjs-foundation/cpc.
+    * Once the final amount spent is known, update the table again with that information.
+    * Certain details of your request (such as the column for date sent to finance) may be unknown initially; these can be filled in and updated during the process of reconciliation.
+
+## Approval
+
+A request is approved when all of the following criteria are met:
+
+* The pull request has been open for at least 72 hours.
+* The request has approval from at least four members of @openjs-foundation/cpc, including a voting representative for your project.
```

Comment:
We should discuss this. One of the current outstanding questions is who qualifies for fund usage. I added a change here, that a voting member for the applicant's project should approve it.

Reply:
Not all projects have voting members.

Reply:
I don't see why a project or its representative should need to positively approve a request, when they do have the opportunity to object to it.

Reply:
ok! I'll go back to earlier rule.